src/XCCDF_POLICY/xccdf_policy.c

```diff
@@ -981,6 +979,52 @@ _xccdf_policy_rule_evaluate(struct xccdf_policy * policy, const struct xccdf_rul
 		return _xccdf_policy_report_rule_result(policy, result, rule, NULL, XCCDF_RESULT_NOT_SELECTED, NULL);
 	}
 
+	bool all_requires_met = true;
```

Comment:
>  I didn't manage to reach xccdf_resolve_rule or xccdf_resolve_group during evaluation.

They were not reached because Rule and Group elements in my Benchmark don't extend any other Rule or Group. 

From specification:
> An XCCDF document that has no @extends or @abstract attributes is called a resolved document.

The Resolve stage is about resolving item inheritance defined in `extends` attribute.

@mpreisler It is possible to do it there, but do we want to  process `requires` and `conflicts` elements at this stage?